boot/freeldr/freeldr/ntldr/wlmemory.c

```diff
@@ -235,14 +235,14 @@ WinLdrSetupMemoryLayout(IN OUT PLOADER_PARAMETER_BLOCK LoaderBlock)
     // Always map the ACPI table if present, otherwise Windows will crash.
     if (Extension->AcpiTable)
     {
-        PVOID AcpiTableClone = MmAllocateMemoryWithType(Extension->AcpiTableSize, LoaderFirmwarePermanent);
+        PVOID AcpiTableClone = MmAllocateMemoryWithType(FindAcpiTableSize(), LoaderFirmwarePermanent);
```

Comment:
We can't use `Extension->AcpiTableSize` here? If `Extension->AcpiTable` is valid, in principle it has been set by this code over here in winldr.c: https://github.com/reactos/reactos/pull/7824/files#diff-9b5cfcb1107667f46f12185ae0b7521d6e73730d3611293d25d2c9432f4edcdcR250
If this is so, `Extension->AcpiTableSize` should already contain the correct value? :thinking:

Reply:
Fixed, The only thing that really needs to be separate is the copy since we have to do it post page table switch.